src/ctim/generators/schemas/judgement_generators.cljc

```diff
@@ -1,7 +1,7 @@
 (ns ctim.generators.schemas.judgement-generators
   (:require [clj-momo.lib.time :as time]
             [clojure.test.check.generators :as gen]
-            [ctim.schemas.common :as schemas-common]
+            [ctim.schemas.shared.common :as schemas-common]
```

Comment:
Do you mean why move common under shared?  I had moved relationships to shared.relationships to avoid confusion with similarly named relationship entity.  When I did that, I asked myself, "What else is shared?" and moved those things as well.